migratecfg/balena-migrate-sample.conf

```diff
@@ -0,0 +1,81 @@
+# Balena migrate example config file
+# copy to /etc/balena-migrate.conf
+# see README.md for instructions
+
+# Path to a resin image to install to the device
+IMAGE_NAME=resin-image-genericx86-64.resinos-2.26.0+rev1-dev-v8.0.0.img.gz
+
+
+# A backup script. Will be invoked by balena-migrate with <scriptname> backup.tgz
+# Top level folders in this archive will be created as volumes (<APPID>_foldername)
+# if the are referenced from application containers
+BACKUP_SCRIPT= # sample-backup.sh
```

Comment:
I would move the default/example values to the line above, it's not very readable, what's set, and what's illustration.

```
# Example:
# BACKUP_SCRIPT="sample-backup.sh"
BACKUP_SCRIPT=
```